tests/base-notebook/test_container_options.py

```diff
@@ -130,206 +78,27 @@ def test_nb_user_change(container: TrackedContainer) -> None:
     ), f"Hidden folder .jupyter was not copied properly to {nb_user} home folder. stat: {output}, expected {expected_output}"
```

Comment:
Sorry that I wasn't clear about this one.

Let's move everything except the hidden folder test (the last part of this function) to the `jupyter-base` tests and keep hidden folder test here.

Reply:
ok - that's what my previous commit did, so I think we're good.  Thanks.

Reply:
I would like you to split `test_nb_user_change` function into two parts.
The first part works perfectly fine in foundation image and should go to the appropriate test folder.
The second one only checks that the `.jupyter` folder is properly copied for base-notebook image and this part of this function stays in `base-notebook` tests.

There will be some small duplication how we start the container, but it's fine.

Reply:
Sorry about that.  Re-reading your original comment - this is what you were getting at.  So there's only a constraint on the names of the test _files_, but not on the names of the test _functions_ themselves?

Reply:
Unfortunately, I don't know the restrictions of pytest 😢 
But I know, that there is an ability to run one particular test in pytest, so I would definitely try to use different function names for different tests.